langchain/package.json

```diff
@@ -457,6 +457,7 @@
     "openai": "^4.41.1",
     "peggy": "^3.0.2",
     "prettier": "^2.8.3",
+    "reflect-metadata": "^0.2.2",
```

Comment:
Is this an accident?

Reply:
Nope! Turns out that to make tests pass, just needed to add that dep